sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -65,11 +67,10 @@
 
 	private CompletionListener initListener;
 	private int state = -1;
-	//public LockScreenConfig lockScreenConfig;
-
+	public LockScreenConfig lockScreenConfig;
```

Comment:
Does the SDL Manager need to hold onto this for it's lifetime? Also why does it need to be public? Won't at least protected suffice?

Reply:
the developer uses the LockScreenConfig to 'build' their lockscreen. It is essentially the instructions to the manager on how to behave:

From `SdlService.java`:

```java
			LockScreenConfig lsc = new LockScreenConfig();
			lsc.setEnabled(true);
                        lsc.setBackgroundColor(R.color.colorPrimary);
			lsc.setShowOEMLogo(true);

			SdlManager.Builder builder = new SdlManager.Builder();
			builder.setAppId(appId);
			builder.setAppName(appName);
			builder.setAppTypes(appType);
			builder.setContext(this);
			builder.setLockScreenConfig(lsc);
			builder.setTransportType(transport);
			proxy = builder.build();
```

Reply:
you are right. set to private, and cleared on proxy start. done.